docs/asciidoc/modules/ROOT/pages/operational/rebind.adoc

```diff
@@ -0,0 +1,84 @@
+APOC provides a list of functions to rebind nodes and relationships:
+
+* `apoc.node.rebind(node)`
+* `apoc.rel.rebind(rel)`
+* `apoc.any.rebind(map/list/paths/...)`
+
+== Why use these functions
+
+Unlike versions up to 3.5, 
+in Neo4j 4.x the entities hold a reference to their originating transaction.
+
+This can cause problems when for example, we create a node (called n1) in a transaction, open a new transaction in which we create another node (called n2) and execute
+via `org.neo4j.graphdb.Node.createRelationshipTo()`,
+that is `n1.createRelationshipTo(n2)`.
+
+
+Is what happens when we perform the following operation:
```

Comment:
Actually this is not the main issue, that also didn't work before, it's regular tx-visibility and I think that will also not be resolved by rebinding.

The issue that all the periodic-iterate tx-state is attached to the originating transaction if the node is not rebound to the new updating tx, which then leads to OOM issues. @vga91